### PR TITLE
feat(#225) - Contact us page for home-spa

### LINKED ISSUE #225: Contact us page for home-spa
## Is your feature request related to a problem? Please describe

Currently, the One Platform does not have a contact us page to view information about the team

## Describe the solution you'd like

Implementation of the following design for contact us page
https://xd.adobe.com/view/efe4d47d-9981-422e-4e7f-d19446acdb31-99d5/screen/45c921ab-71e8-45d3-bd81-22e0ab4a0616/One-portal-contact-us-

## Describe alternatives you've considered

None

### PR DESCRIPTION
**Pull request by hybridx**
feat(#225) - Added contact-us page to home-spa and teamMembers Array

## Does this PR introduce a breaking change

No

## Screenshots

Screenshot_2020-04-29 Contact Us One Platform

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

**Comment by riginoommen** [comment]
If i am not wrong we discussed about using the actual profile thumbnalis in the page.

File: packages/home-spa/src/mocks/stub.js [hunk 1]
```diff
@@ -109,4 +109,47 @@ export const microserviceMock = [
       ],
       illustration: 'img/User-profile-illustration.svg',
    },
-];
\ No newline at end of file
+];
+
+export const teamMembers = [
```
Comment (deshmukhmayur) [inline packages/home-spa/src/mocks/stub.js:114]: Can you get this data from the Rover Group? `one-portal-devel`.
And even the profile pictures.

Comment (hybridx) [reply packages/home-spa/src/mocks/stub.js:114]: @deshmukhmayur haven't we already discussed fetching rover information after the MVP @deshmukhmayur @npatil9 @gisumit

Comment (deshmukhmayur) [reply packages/home-spa/src/mocks/stub.js:114]: Oh yes we did. I forgot about it.
Nevermind my comment.

**Pull request merged**